chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies

```diff
@@ -1,5 +1 @@
 node-gyp-headers
```

Comment:
This was relevant when we were trying to build without an internet connection. 

This needs to be removed and the coorresponding dir in CE also needs removed (assume the later will just go when CE does).